mailserver/mailserver.go

```diff
@@ -17,28 +17,30 @@
 package mailserver
 
 import (
+	"crypto/ecdsa"
 	"encoding/binary"
 	"errors"
 	"fmt"
 	"math/rand"
+	"reflect"
 	"sync"
-
 	"time"
 
 	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/log"
 	"github.com/ethereum/go-ethereum/rlp"
+	prom "github.com/prometheus/client_golang/prometheus"
+	gethbridge "github.com/status-im/status-go/eth-node/bridge/geth"
+	"github.com/status-im/status-go/eth-node/crypto"
 	"github.com/status-im/status-go/eth-node/types"
 	"github.com/status-im/status-go/params"
+	"github.com/status-im/status-go/waku"
 	"github.com/status-im/status-go/whisper/v6"
-
-	prom "github.com/prometheus/client_golang/prometheus"
 )
 
 const (
 	maxQueryRange = 24 * time.Hour
-	defaultLimit  = 2000
+	defaultLimit  = 1000
```

Comment:
Maybe a comment explaining purpose of this limit? Also, the word "default" doesn't really describe well what this does, maybe renaming it would be a good idea.